### PR TITLE
Build wasm runtime on CI and upload it as artifact

### PR DESCRIPTION
**Pull request by geigerzaehler**
As preparation for #492 we start building the wasm runtime on CI and uploading it as an artifact.

### CODE REVIEW

File: ci/run [hunk 1]
```diff
@@ -43,6 +43,8 @@ time cargo fmt --all -- --check
 
 export RUSTFLAGS="-D warnings"
 
+touch runtime/latest.wasm
```
Comment (NunoAlexandre) [inline ci/run:46]: is this necessary to bypass some caching?

Comment (geigerzaehler) [reply ci/run:46]: Good catch! This is just an artifact from a previous experiment. Will remove it.

**Comment by geigerzaehler** [comment]
@NunoAlexandre: could you have another look?

**Comment by geigerzaehler** [comment]
> Is this useful in feature branches? Do we have a use case for those cases? If not, I'd prefer to only run this on the master branch to avoid increasing the build time.

It will become necessary to run this on every build once we remove `runtime/latest.wasm` from the repo (see #492). But at the moment this is not necessary and serves merely as a proof-of-concept. I will limit this to master branch builds.

**Pull request merged**